src/com/company/Game.java

```diff
@@ -0,0 +1,258 @@
+package com.company;
+
+import java.util.ArrayList;
+
+public class Game {
+
+    ArrayList<Team> teams= new ArrayList<>();
+    int overs = 6;
+    public void setupTeams(){
+        Team team1 = new Team();
+        Team team2 = new Team();
+        team1.setId(1001);
+        team2.setId(1002);
+        team1.setName("CSK");
+        team2.setName("MI");
+        team1.setPlayers(11) ;
+        team2.setPlayers(11) ;
+        team1.setNotOut(11);
+        team2.setNotOut(11);
+        System.out.println("Team 1 name: " + team1.getName());
+        System.out.println("Team 2 name: " + team2.getName());
+        team1.setupPlayers(101);
+        team2.setupPlayers(201);
+        teams.add(team1);
+        teams.add(team2);
+
+    }
+
+
+
+    public void match()
+    {
+        //int team1_w = team1.getPlayers(), team2_w = team2.getPlayers();
+        int totalBalls = overs*6;
+        int tossVar = toss();
+        if(tossVar==1)
```

Comment:
maintain two variables battingTeam and bowlingTeam and swap them after an inning.

Reply:
Great point. Deleted the ArrayList which was storing both teams and changed the code that directly access batting and bowling team, as per your suggestion.